altair/vegalite/v2/examples/scatter_linked_brush.py

```diff
@@ -0,0 +1,25 @@
+"""
+Faceted Scatter Plot with Linked Brushing
+-----------------------------------------
+This is an example of using an interval selection to control the color of
+points across multiple facets.
+"""
+
+import altair as alt
+from vega_datasets import data
+
+cars = data.cars()
+
+brush = alt.selection(type='interval', resolve='global')
+
+base = alt.Chart(cars).mark_point().encode(
+    y='Miles_per_Gallon',
+    color=brush.condition('Origin',
```

Comment:
Seeing this syntax for condition with selection as predicate, I wonder how would condition works for general `test` predicate (see bottom of https://vega.github.io/vega-lite/docs/condition.html). 

I wonder if we should make a general construct that works for both `test` and `selection`? 

For example:

```
color=alt.condition(brush, "Origin", alt.ColorValue('gray'))
```
where `brush` is a selection could work

or we can put a test predicate (i'm not sure if there is a specific Altair syntax for it, but let's say it can support Vega expression string as well). 
```
color=alt.condition("datum.x > 5", "Origin", alt.ColorValue('gray'))
```

Reply:
I really like this.

Reply:
OK, changed!